### ISSUE TITLE
Астрологи

### ISSUE THREAD

**Issue by YogurtTheHorse** [yegorf1/RogueBot#221]
Каждый день давать один из бонусов в зависимости от _гороскопа_ (Гороскоп определяется при создании перса):
- 1.5x бабла
- 1.5x дамага
- 1.5х инты
- etc

Придумать смешные фразочки

**Comment by Fadds** [comment]
Не реализовывать, пока не придумаем >12 равносильных бонусов и 12 Знаков.

> Жаль, что ты не родился, когда Луна была в Водолее, а Марс в Венере. Тем не менее, ты родился. Твой Знак Клоака:

---

> Неточные Весы
> Скорпион-Сабзиро
> Лизнецы
> Рак Тельца

Добейте до 12

**Comment by thekiba** [comment]
Только нужно не при создании персонажа выдавать, а то начнут спамить с помощью /start.

Предлагаю ввести какой-нибудь квест на серединке, чтобы там выдавался такой бонус. И еще желательно, чтобы бонус через какое-то время менялся.

**Comment by Fadds** [comment]
Да, сутки — очень много. Да и бонус, по-моему, сладковат. *1.25?

**Comment by YogurtTheHorse** [comment]
> сладковат

Ну это все можно корректировать

**Comment by YogurtTheHorse** [comment]
> 12 Знаков

Я змееносец и я буду жаловаться.
